pytorch_lightning/core/lightning.py

```diff
@@ -1794,11 +1799,39 @@ def to_onnx(
 
         input_sample = self._apply_batch_transfer_handler(input_sample)
 
+        if not isinstance(input_sample, (Tuple, list)):
```

Comment:
```suggestion
        if not isinstance(input_sample, (tuple, list)):
```
or Tuple, List?

Reply:
`(tuple, list)` is the right one, guessing that was a typo from @vballoli to use `typing.Tuple` instead of the `tuple` class